xml/libvirt_guest_installation.xml

```diff
@@ -508,25 +508,37 @@ network=vnet_nated</screen>
         <term>Using &uefisecboot;</term>
         <listitem>
           <para>
-            Install OVMF as described in
-            <xref
-      linkend="sec-vt-installation-ovmf" />. Then add the
+            Install OVMF and become familiar with manual verses automatic
+            firmware selection as described in
```

Comment:
s,verses,versus,

but here you suggest to become familiar with "manual versus automatic firmware selection" as described in "sec-bt-installation-ovmf", but there is no such clearly named section there.

This information is present currently in section "sec-vt-installation-ovmf", just as an unnamed paragraph:

    <para>                                                                                                                                  
      The firmware used by virtual machines is auto-selected. The                                                                           
      auto-selection is based on the JSON files in the firmware package                                                                     
      described above. 
    </para>

   (note that for the "above" there I missed the reference just in the previous paragraph and started scattering a search in the preceding text. This could be avoided by inverting the order of the paragraphs and clearly labeling them with a title).

I'd also suggest that if we install as I think we do qemu-ovmf-x86_64 by default when using the patterns or yast installation, it is more of a "configuration" of the vm rather than an installation,
so the way I would have called this paragraph would have been "Configuring UEFI support for guests" rather than "Installing UEFI support". We should document I think with the yast and patterns in mind, the users that choose their own packages can skip to the "manual" portion of the documentation.

The info about the default path we want the user to take should be first and prominent, privileging auto-configuration and defaults, and the "advanced", "manual" or "explanatory" sections where we go in painful detail to explain all different variants of firmware could be separate at the end. Most people do not and should not care, and going through a wall of dense information is very painful for the reader.

F.e:

  <sect1 xml:id="sec-vt-installation-ovmf">
    <title>Configuring UEFI support for guests</title>

    <paragraph about KVM vs Xen>

    <subsection something>Configure UEFI support for guests using automatic firmware selection (or firmware auto-selection maybe, but then also correct the reference. People will CTRL+F search the text to find the section)</subsection>

... explain how to get things done with auto-selection ...

note: in the example we put secure = on, is it possible to just put secure=off there and get UEFI without secure boot?

<subsection>
Advanced manual configuration

Here explain how autodetection works, json files, how to configure manually etc.

All this complexity and explanation is a lot of useless chatter for someone that wants to get __ done,
just install the kvm pattern or with yast, and trying to get autoselection for UEFI.